ui/app/scripts/shared/views/about.html

```diff
@@ -30,7 +30,7 @@
 <h2>Get the Spring Cloud Data Flow Shell</h2>
 
 <p class="index-page--subtitle">As an alternative to the Dashboard UI, you can also
-   <a href="https://repo.spring.io/libs-snapshot/org/springframework/cloud/spring-cloud-dataflow-shell/{{dataflowVersionInfo.app.version}}/spring-cloud-dataflow-shell-{{dataflowVersionInfo.app.version}}.jar"
+   <a href="https://repo.spring.io/libs-snapshot/org/springframework/cloud/spring-cloud-dataflow-shell/{{dataflowVersionInfo.versionInfo.core.version}}/{{dataflowVersionInfo.versionInfo.core.version}}.jar"
```

Comment:
looks like we lost the  `spring-cloud-dataflow-shell-` part of the link, will re-add on merge